README.md

```diff
@@ -1,6 +1,48 @@
-# apphub
+# AppHub 
 
-## Project setup
+[![Known Vulnerabilities](https://snyk.io/test/github/esdc-devx/AppHub/badge.svg?targetFile=app-hub-api%2Fpackage.json)](https://snyk.io/test/github/esdc-devx/AppHub?targetFile=app-hub-api%2Fpackage.json)
+[![Build Status](https://dev.azure.com/dx-team/AppHub/_apis/build/status/esdc-devx.AppHub?branchName=master)](https://dev.azure.com/dx-team/AppHub/_build/latest?definitionId=2&branchName=master)
+
+A dashboard for displaying information on systems department wide. 
+
+## Run Integration and a11y Tests
+
+Run the following command in the `integration-tests` folder. 
+
+### Setup Tests
+```
+npm ci
```

Comment:
`npm ci` is used for running tests in a CI Environment which is what these tests are for, I figured it would make more sense to use the same command locally as in our CI Environment.

https://docs.npmjs.com/cli/ci.html

Reply:
cool. When i tried it the first time it didn't work.